xdsl/interactive/app.py

```diff
@@ -147,8 +178,24 @@ def update_current_module(self) -> None:
             pipeline = PipelinePass([p() for p in self.current_selected_pass_list])
             pipeline.apply(ctx, module)
             self.current_module = module
+            # for current_module's that are valid ModuleOp's find the condensed pass list
```

Comment:
```suggestion
```

I think the comment is a bit disorienting; since we are in the `try` of a `try/catch` block, let's just assume `module` has been parsed successfully and just follow the happy path scenario.